app.rb

```diff
@@ -91,4 +98,32 @@ def list_rentals_for_person
       puts "Date: #{rental.date}, Book: #{rental.book.title}"
     end
   end
+
+  def run # rubocop:disable Metrics/MethodLength, Metrics/CyclomaticComplexity
+    loop do
+      display_options
+      option = gets.chomp.to_i
+      if option == 7
+        puts 'Thank you for using the app. Goodbye!'
+        break
+      elsif option.between?(1, 6)
+        case option
+        when 1
+          list_books
+        when 2
+          list_people
+        when 3
+          create_person
+        when 4
+          create_book
+        when 5
+          create_rental
```

Comment:
- [ ] Great job so far! 😊 However, I've noticed a potential issue that could impact the functionality of the "List Person" feature. In the video provided in the project requirements, you can see that when displaying the list of persons, they are listed with an `index` number. This `index` number is crucial because it allows users to easily select a specific person when creating a rental. 

  - In your current implementation, this `index` number is missing, which could confuse users and make it challenging for them to specify who they want to create a rental for. To enhance the user experience and align your app with the project requirements, please consider adding the `index` number when displaying the list of persons. See below for more details:
  <table>
<tr>
<th align=“middle”>Main UI</th>
<th align=“middle”>Your UI</th>
</tr>
<tr>
<td>
<img width="347" alt="Screenshot 2023-10-09 at 21 04 33" src="https://github.com/BenMKT/OOP-school-library/assets/107192076/14ffe5ce-b5a9-49ca-b5ba-d86b2131fd68">
</td>
<td>
<img width="347" alt="Screenshot 2023-10-09 at 21 01 16" src="https://github.com/BenMKT/OOP-school-library/assets/107192076/67632f07-0a29-4e1e-8a71-80a6f9911e77">
</td>
</tr>
</table>

This small adjustment can make a significant difference in user clarity and usability. Keep up the good work! 😊👍

Reply:
- Modify `list_people` method by adding the index number when displaying the list of persons.